src/components/interaction-designer/block-types/MobilePrimitives_SelectManyResponseBlock.vue

```diff
@@ -61,30 +78,44 @@ import BlockId from '../block-editors/BlockId.vue'
 import SelectOneResponseBlock from './MobilePrimitives_SelectOneResponseBlock.vue'
 import GenericContactPropertyEditor from '../block-editors/GenericContactPropertyEditor.vue'
 
+const flowVuexNamespace = namespace('flow')
 const blockVuexNamespace = namespace(`flow/${BLOCK_TYPE}`)
 const builderVuexNamespace = namespace('builder')
 
 @Component({
   components: {
     GenericContactPropertyEditor,
-    BlockNameEditor,
+    ResourceVariantTextEditor,
+    BlockExitSemanticLabelEditor,
+    BlockId,
     BlockLabelEditor,
+    BlockNameEditor,
+    BlockOutputBranchingConfig,
     BlockSemanticLabelEditor,
-    BlockExitSemanticLabelEditor,
+    ChoicesBuilder,
     FirstBlockEditorButton,
     ResourceEditor,
-    BlockId,
     Categorization,
   },
 })
 export class MobilePrimitives_SelectManyResponseBlock extends SelectOneResponseBlock {
+  @Prop() readonly declare block: ISelectOneResponseBlock
+
+  @Prop() readonly declare flow: IFlow
+
   showSemanticLabel = false
 
+  SupportedMode = SupportedMode
+  findOrGenerateStubbedVariantOn = findOrGenerateStubbedVariantOn
+
   //Important: Even we extends from SelectOneResponseBlock, to avoid conflict
   // we SHOULD re-declare @blockVuexNamespace based getter, state, action, mutation
+  @flowVuexNamespace.Getter declare resourcesByUuid: { [key: string]: IResource }
```

Comment:
@bzabos , I think we don't need to redeclare `@flowVuexNamespace` Getter & Action here, because the namespace hasn't changed from SelectOne.

Could you pls remove them and test if you implementation still work ?

Reply:
Oooh! I think I understand now 😅  It works, thanks!